packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -160,6 +160,14 @@ export const render = async (req, res, next) => {
     )
     console.log('--- serialized extensions in react-rendering', serializedExtensions)
 
+    applicationExtensions.forEach((extension) => {
+        const componentMap = extension.getComponentMap(_routes)
+        console.log('JINSU componentMap', componentMap)
+        extension.deserialize(componentMap)
+    })
+
+    let routes = await getAllRoutes(res.locals)
```

Comment:
we need to deserialize also on the server because after we route match the `component` of the current route is passed to the initial App state.